Makefile

```diff
@@ -301,7 +302,7 @@ dev-push: docker-build helm-push
 
 .PHONY: dev-templates
 dev-templates: templates-generate
-	$(KUBECTL) -n $(NAMESPACE) apply -f $(PROVIDER_TEMPLATES_DIR)/hmc-templates/files/templates
+	$(KUBECTL) -n $(NAMESPACE) apply --force -f $(PROVIDER_TEMPLATES_DIR)/hmc-templates/files/templates
```

Comment:
To help support the `NO_CLEANUP` use case and make the tests work if things already exist without failure.  Some of the template specs are immutable, iirc `--force` replaces the spec instead of a standard apply.

Reply:
Ok, got it. Let's leave it here